packages/animations/example/lib/container_transition.dart

```diff
@@ -102,7 +102,10 @@ class _OpenContainerTransformDemoState
 
   @override
   Widget build(BuildContext context) {
+    final GlobalKey<ScaffoldState> scaffoldKey = GlobalKey<ScaffoldState>();
```

Comment:
The global key should be defined outside of the build method. Otherwise, a new global key is created every time the widget builds, which is not desirable.